webpack.config.node.js

```diff
@@ -73,7 +73,8 @@ module.exports = {
 	},
 	plugins: [
 		// Require source-map-support at the top, so we get source maps for the bundle
-		new webpack.BannerPlugin( 'require( "source-map-support" ).install();', { raw: true, entryOnly: false } )
+		new webpack.BannerPlugin( 'require( "source-map-support" ).install();', { raw: true, entryOnly: false } ),
+		new webpack.NormalModuleReplacementPlugin( /^analytics$/, 'lodash/utility/noop' ),
```

Comment:
>  forget what's requiring analytics here

Nothing AFAICS -- was a leftover from the spike that also had the showcase SSR'ed. Reverted that commit since the masterbar doesn't seem to do any analytics.